inc/composer/class-docker-compose-generator.php

```diff
@@ -91,7 +91,7 @@ protected function get_php_reusable() : array {
 					'condition' => 'service_started',
 				],
 			],
-			'image' => 'humanmade/altis-local-server-php:4.0.0-dev',
+			'image' => 'humanmade/altis-local-server-php:4.2.0',
```

Comment:
@roborourke `-dev` IIRC is the dev container, not a "dev version" that has xdebug intsalled: https://github.com/humanmade/docker-wordpress-php/blob/master/Dockerfile-dev. Did we stop using `-dev` just for the tag (and it's built from `-dev`?)-- sorry I can't remember but wanted to check, as I think it might have even been me who made that change!

Reply:
Ah gotcha, yeah so the GitHub workflow uses the `Dockerfile-dev` file to build. It wasn't a dev container - it was always just used to reference the version that had Xdebug installed. I think that's fine personally as Xdebug 3 can be present and have minimal impact. Plus we don't use the builds on Docker hub on our infrastructure.

Reply:
I didn't make that particular change, I just didn't notice it earlier - I thought we still had the regular tag and tag + `-dev` with Xdebug being built and pushed up.

The `-dev` suffix was previously how we toggled between a container with Xdebug and one without, but we use the `XDEBUG_MODE` setting to switch it on or off now.